test/fixtures/resource-discovery/widgets.yml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: widgets.api.foobar.com
+  annotations:
+    kubernetes-deploy.shopify.io/prunable: "true"
```

Comment:
Should we have a third one that has the annotation, but set to "false"?

Reply:
I think its overkill, we already test the case where its not pruned.